docs/conf.py

```diff
@@ -80,9 +58,9 @@ def get_short_version():
 # built documents.
 #
 # The short X.Y version.
-version = get_short_version()
+version = objgraph.__version__
```

Comment:
I may be misunderstanding Sphinx here, but shouldn't this be truncated to two components?  `__version__` may also include a bugfix component X.Y.Z.

Reply:
In documentation I think that should be included full version.

Reply:
Then why is there a separate `release` setting with the full version?